keyboards/ckeys/thedora/readme.md

```diff
@@ -32,7 +32,15 @@ All cKeys PCBs are open source. However, this one has not been posted yet becaus
 
 If you want to reflash the pre-installed firmware, use the `.bin` file for Proton C. Flash with the QMK Toolbox or via the command line.
 
-Building for Proton C: `make ckeys/thedora:default`
+**Building for Proton C:**
+`make ckeys/thedora:default`
+
+**Enter DFU-Bootloader Mode:**
```

Comment:
`Enter DFU-Bootloader Mode` is what the device does, not the host, so I would say this is bit misleading.

The command here is only host side. It starts the process of searching for a bootloader device to flash, it doesnt actually trigger the device to go into the bootloader.

Reply:
> Im not the best person for wordsmithing suggestions, but I think the terminology here is mixed up.

Ah thanks for the input, admittedly I am still learning the lexicon on how this works and I cribbed this description of the process from another keyboard's site that uses the same microcontroller, since this was not very elucidating. Hoping I can save someone else an hour of their time researching :)